data_gen/data_generation/writers/datefilters.py

```diff
@@ -0,0 +1,81 @@
+"""Create output filters for data values.
```

Comment:
This file was renamed to reflect that it includes date filters only. And the filters from the SBAC folder was moved into this folder.